tests/at_tests.cpp

```diff
@@ -26,39 +26,39 @@ SUITE(at_tests)
 {
     TEST(static_array)
     {
-        int a[] = { 1, 2, 3, 4 };
+        size_t a[] = { 1, 2, 3, 4 };
 
-        for (int i = 0; i < 4; ++i)
+        for (size_t i = 0; i < 4; ++i)
             CHECK(at(a, i) == i+1);
```

Comment:
I'm with @gdr-at-ms ... why does `at()` take `size_t`? It should not. It should take `ptrdiff_t`. If that's my fault, it's probably because we never updated it after the original prototype back in the mists of the past. But that should change, rather than be worked around.